tempesta_fw/t/unit/test_hash.c

```diff
@@ -107,8 +106,8 @@ TEST(tfw_hash_str, hashes_all_chars)
 	unsigned long h1, h2;
 	char buf1[256] = { 0 };
 	char buf2[256] = { 0 };
-	TfwStr s1 = { .len = 0,	.ptr = buf1 };
-	TfwStr s2 = { .len = 0, .ptr = buf2 };
+	TfwStr s1 = { .len = 0,	.data = buf1 };
+	TfwStr s2 = { .len = 0, .data = buf2 };
```

Comment:
Well, yes, just `{ .data = buf }` is sufficient here. All other members will be initialized to zero values.